test/cases/no_compile_panic_for_arith_wrap.zig

```diff
@@ -0,0 +1,12 @@
+pub const panic = @compileError("");
+
+export fn entry() usize {
+    var x: usize = 0;
+    x +%= 1;
+    x -%= 1;
+    x *%= 1;
```

Comment:
This should test saturating arithmetic too.

Reply:
All tags for unchecked operators break from the safety block on the same condition. Am I missing something obvious?

Reply:
I came up with the list of operators to test by looking at all the calls to `analyzeArithmetic`, and picking the unchecked ones.  Using the current implementation of a function to justify the completeness of a test is not really a great way to write tests, rather the test should test exactly the operators that we don't want to panic, to ensure that none of them regress in the future.  I would also argue that `x *= 1;` could be changed in the future to never panic, and so `x *%= 2;` is a more future-proof test.